pypeit/spectrographs/not_alfosc.py

```diff
@@ -269,3 +284,85 @@ def config_specific_par(self, scifile, inp_par=None):
         # Return
         return par
 
+
+
+
+class NOTALFOSCSpectrographVert(NOTALFOSCSpectrograph):
+    """
+    Child to handle Vertical slits for NOT ALFOSC spectrograph
+    """
+    ndet = 1
```

Comment:
Most of these will be inherited (I think?!). I think the only information that needs to be updated here is:
```
    name = 'not_alfosc_vert'
    comment = 'Grisms 4, 8, 17, 19. For vertical slits only.'
```